cdap-ui/app/hydrator/services/canvas-factory.js

```diff
@@ -106,6 +106,80 @@
       return finalConnections.map(function (conn) { delete conn.visited; return conn; });
     }
 
+    function orderConditionAlertErrorConnections(connections, conditionNodes, alertNodes, errorNodes) {
+      let swappedConnections = false;
```

Comment:
nit: This looks a little clunky. Can't we just send back an array of connections that we allways set to jsplumb whenever the user cleans up the graph? is it hitting performance badly?